x/nexus/exported/types.go

```diff
@@ -6,6 +6,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// Handler defines a function that handles address verification upon a request to link addresses
+type Handler func(ctx sdk.Context, address CrossChainAddress) error
+
```

Comment:
can we give this a more specific name than `Handler`?

Reply:
renamed it to `AddressValidator`